packages/demo-site/remix.config.js

```diff
@@ -2,7 +2,7 @@
 module.exports = {
   ignoredRouteFiles: ['**/.*'],
   server: process.env.NETLIFY || process.env.NETLIFY_LOCAL ? './server.ts' : undefined,
-  serverBuildPath: '../../.netlify/functions-internal/server.js',
+  serverBuildPath: './.netlify/functions-internal/server.js',
```

Comment:
With the improved monorepo support, the `serverBuildPath` will build in the `packages/demo-site` folder which is where we look for the `.netlify` folder in a monorepo project.